engine/apps/twilioapp/migrations/0003_auto_20230408_0711.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.18 on 2023-04-08 07:11
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('twilioapp', '0002_auto_20220604_1008'),
+    ]
+
+    database_operations = [
+        migrations.AlterModelTable('PhoneCall', 'phone_notifications_oncallphonecall'),
+        migrations.AlterModelTable('SMSMessage', 'phone_notifications_oncallsms')
+    ]
+
+    state_operations = [
+        migrations.DeleteModel('PhoneCall'),
+        migrations.DeleteModel('SMSMessage')
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=database_operations,
+            state_operations=state_operations
+        )
+    ]
```

Comment:
Consider using https://docs.djangoproject.com/en/3.2/ref/models/options/#db-table instead of renaming tables to avoid data migrations